esmvaltool/diag_scripts/mpqb/mpqb_lineplot_growthrate.py

```diff
@@ -0,0 +1,151 @@
+            'crezee_bas',
+            'hassler_birgit',
+        ],
+        'projects': ['cmug'],
+        'references': [
+            'acknow_project',
+        ],
+    }
+    return record
+
+
+def _unify_time_coord(cube):
+    """Unify time coordinate of cube."""
+    if not cube.coords('time', dim_coords=True):
+        return
+    time_coord = cube.coord('time')
+    dates_points = time_coord.units.num2date(time_coord.points)
+    dates_bounds = time_coord.units.num2date(time_coord.bounds)
+    new_units = Unit('days since 1850-01-01 00:00:00')
+    new_time_coord = iris.coords.DimCoord(
+        new_units.date2num(dates_points),
+        bounds=new_units.date2num(dates_bounds),
+        var_name='time',
+        standard_name='time',
+        long_name='time',
+        units=new_units,
+    )
+    coord_dims = cube.coord_dims('time')
+    cube.remove_coord('time')
+    cube.add_dim_coord(new_time_coord, coord_dims)
+
+
+def main(cfg):
+    """Create lineplot."""
+    # Limits for the y-axis
+    ylims = [-5., 15.]
+
+    # Get a description of the preprocessed data that we will use as input.
+    input_data = cfg['input_data'].values()
+
+    grouped_input_data = group_metadata(input_data, 'alias', sort='alias')
+
+    logger.info(
+        "Example of how to group and sort input data by standard_name:"
+        "\n%s", pformat(grouped_input_data))
+
+    plt.clf()
+    fig, (ax1, lax) = plt.subplots(nrows=2,
+                                   gridspec_kw={"height_ratios": [10, 1]},
+                                   figsize=(10, 5))
+
+    plt.sca(ax1)
+    for dataset in grouped_input_data:
+        dataset_cfg = grouped_input_data[dataset][0]
+        alias = dataset_cfg['alias']
+
+        logger.info("Opening dataset: %s", dataset)
+        cube = iris.load_cube(dataset_cfg['filename'])
+        _unify_time_coord(cube)
+
+        # caluclate the growth rates
+        diff_data = np.diff(cube.data)
+        cube = cube[1:]
+        cube.data = diff_data
+
+        # Set default if not defined.
+        label = get_mpqb_cfg('datasetname', alias)
+        color = get_mpqb_cfg('datasetcolor', alias)
+
+        # iris.quickplot.plot(cube, label=label, color=color,
+        #                     linestyle='dotted')
+        # iris.quickplot.plot(cube, label=label, color=color,
+        #                     linestyle='dashed')
+        iris.quickplot.plot(cube, label=label, color=color)
+    plt.xticks(rotation=90)
+    # Add the zero line
+    plt.axhline(y=0, linestyle=':', color='k')
+    plt.tight_layout()
+
+    # Time axis formatting
+    years = mdates.YearLocator()  # every year
+    years_fmt = mdates.DateFormatter('%Y')
+    ax1 = plt.gca()
+    ax1.xaxis.set_major_locator(years)
+    ax1.xaxis.set_major_formatter(years_fmt)
+    ax1.grid(True, which='major', axis='x')
+
+    ax1.set_ylim(ylims)
+    ax1.set_ylabel('Change XCH4 (ppbv)')
+    ax1.set_title('Time series of annual XCH4 growth rate')
```

Comment:
```suggestion
    ax1.set_ylabel(f"Change in {cube.var_name.upper()} ({cube.units})")
    ax1.set_title(f"Time series of annual {cube.var_name.upper()} growth rate")
```

Reply:
Also here, thanks for the suggestions, they are great.